superset/assets/javascripts/SqlLab/components/App.jsx

```diff
@@ -32,11 +32,16 @@ class App extends React.PureComponent {
     this.setState({ hash: window.location.hash });
   }
   getHeight() {
-    const navHeight = 90;
-    const headerHeight = $('.nav-tabs').outerHeight() ?
-      $('.nav-tabs').outerHeight() : $('#search-header').outerHeight();
-    const warningHeight = $('#navbar-warning').outerHeight();
-    const alertHeight = $('#sqllab-alerts').outerHeight();
+    const warningEl = $('#navbar-warning');
```

Comment:
[optional] would it make more sense to put most of this stuff in one `div` and look for the `outerHeight` of that div. Or maybe put a common CSS class on all of the and summing `outerHeight` over that simple selector?